inc/components/class-component.php

```diff
@@ -458,6 +457,17 @@ function ( $child ) {
 			}
 		}
 
+		/**
+		 * Filter the children values for all instances of this component.
+		 *
+		 * @todo Decide how this filter should be approached. This is a
+		 *       temporary solution that requires discussion.
+		 *
+		 * @param array $children Children for this component.
+		 * @param array $config   Config for this component.
+		 */
+		$children = apply_filters( 'wp_irving_component_children_' . $this->get_name(), $children, $this->config );
```

Comment:
I think I'd rather have a general use filter here that passes the component name as a parameter rather than a dynamic one, sine this would result in integrators writing code that looks like this, which is fairly unorthodox: 

```
add_filters( 'wp_irving_component_chidren_irving/post-list', function () { ... } );
```

Reply:
Yeah, we need to discuss and figure out a plan of attack. Do you have any thoughts on it right now?

Reply:
I'd do exactly what you're doing, but name it generically, `wp_irving_component_children` with three params: `array $children, array $config, string, $name`. Placement of the filter is exactly where I was going to put it.